fix_pmd.py

```
@@ -0,0 +1,19 @@
+with open('.github/workflows/pmd.yml', 'r') as f:
```

Ruff (0.15.9)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)